package.json

```diff
@@ -190,22 +190,24 @@
     "karma-ie-launcher": "0.2.0",
     "karma-mocha": "0.2.0",
     "karma-safari-launcher": "0.1.1",
-    "license-checker": "3.1.0",
-    "load-grunt-config": "0.19.1",
+    "license-checker": "5.1.2",
+    "load-grunt-config": "0.19.2",
     "makelogs": "3.0.2",
-    "mocha": "2.3.0",
+    "marked-text-renderer": "0.1.0",
+    "mocha": "2.5.3",
     "ncp": "2.0.0",
     "nock": "2.10.0",
-    "npm": "2.15.8",
+    "npm": "3.10.6",
     "portscanner": "1.0.0",
     "simple-git": "1.37.0",
     "sinon": "1.17.2",
-    "source-map": "0.4.4",
-    "source-map-support": "0.4.0",
+    "source-map": "0.5.6",
+    "source-map-support": "0.2.10",
+    "supertest": "1.2.0",
     "supertest-as-promised": "2.0.2"
   },
   "engines": {
-    "node": "4.4.7",
-    "npm": "2.15.8"
+    "node": "6.4.0",
+    "npm": "3.10.6"
```

Comment:
We should use whatever version of npm comes with the given version of node rather than the latest available.  I think 3.10.3 is what ships with 6.4.0.